index.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Tangerine">
+          <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
+          <link href="https://fonts.googleapis.com/icon?family=Material+Symbols"
+          rel="stylesheet">
+  </head>
+  <body>
+    <section class="whole-main">
+    <section>
+      <div class="contaniner">
+        <!-- //card 1 -->
+        <div class="card">
+            <a href="">
+          <!-- Card Image  -->
+          <div class="img">
+            <img src="./Pictures/Snapshoot Portfolio.png" alt="" />
+          </div>
+          <div class="card-main">
+            <!-- Card main heading -->
+            <div class="card-name-h1">
+              <h1>Tonic</h1>
+            </div>
+            <div class="experience">
+              <!-- card Frame -->
+              <ul>
+                <li class="a">CANOPY</li>
+                <li class="b">Back End Dev</li>
+                <li class="c">2015</li>
+              </ul>
+            </div>
+            <div class="parga">
+              <!-- card Des -->
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="language">
+              <!-- card Language  -->
+              <ul>
+                <li class="a l-a">html</li>
+                <li class="b">css</li>
+                <li class="c">javascript</li>
+              </ul>
+            </div>
+            <div class="button">
+              <!-- card button  -->
+              <button>See project</button>
+            </div>
+          </div>
+        </a>
+        </div>
```

Comment:
- I would like to kindly suggest that you follow the style guidelines provided in the Figma template to style each card. 

- The fonts, spacing, margins, padding, background, buttons, and lists need to be reworked on. 
 I have attached a link, I think you can refer to the [Project Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) for this task.  🌸 

| Figma Template Design  |Your Design |
| :---:   | :---: | 
| ![goodpc](https://user-images.githubusercontent.com/99246359/198838516-46832a72-5511-4693-b2ee-adba0857610c.PNG) |![badpc](https://user-images.githubusercontent.com/99246359/198838604-deaad5dd-fc9a-4fc0-b8dc-a3bf26bdb62f.PNG)  |

Reply:
Sorry I just upload the wrong style.css I have updated Now You can check it. Also the error is opacity because opacity has no unit. 
This the project 
![image](https://user-images.githubusercontent.com/80938128/198839723-bfd84230-0c05-4f46-9aa0-62d4307d3700.png)